src/targets/Logary.Targets.Stackdriver/App.config

```diff
@@ -12,6 +12,21 @@
     <gcServer enabled="true" />
 
   <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+  <dependentAssembly>
```

Comment:
Please remove this change to the StackDriver taget.

Reply:
done.